projects/plugins/crm/src/entities/class-invoice.php

```diff
@@ -0,0 +1,310 @@
+<?php
+/**
+ * Invoice Entity.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Entities;
+
+/**
+ * Invoice class.
+ *
+ * @since $$next-version$$
+ */
+class Invoice {
+
+	/**
+	 * The DB ID of the object.
+	 *
+	 * @var int
+	 */
+	public $id = -1;
```

Comment:
Probably not worth changing here since we're just matching DAL, but I really want to move away from -1 meaning "not set".

Reply:
Yes, in this case, we are mirroring the model from DAL. So we have to continue using -1.